internal/core/attribute.cpp

```diff
@@ -60,22 +62,27 @@ float MaxAbs::compute(
     return std::abs(max);
 }
 
+} // namespace
+
+float MaxAbs::compute(
+    ResampledSegment const & segment
+) noexcept (false) {
+    return maxAbs(segment);
+}
+
 float MaxAbsAt::compute(
     ResampledSegment const & segment
 ) noexcept (false) {
 
-    auto max_abs_val = *std::max_element(segment.begin(), segment.end(),
-    [](const double& a, const double& b) {
-            return std::abs(a) < std::abs(b);
-        }
-    );
+    auto max_abs_val = maxAbs(segment);
     auto max_abs_index = std::distance(
         segment.begin(),
         std::find_if(segment.begin(), segment.end(),
-            [&](const double& val) {
+        [&](const double& val) {
                 return std::abs(val) == std::abs(max_abs_val);
```

Comment:
Do you need to take std:abs from max_abs_val. Don't you already return std::abs ?

Reply:
yes, fixed
https://github.com/equinor/vds-slice/commit/15e91b991d3ddd90e96058fbbb5769827b1c2f08#diff-0a24dd0852852a74dc874fdfcbb7826193dfa983b4b02f34f87d06fb28e91a40R82